app/models/user.rb

```diff
@@ -44,7 +44,15 @@ class User < ApplicationRecord
   has_one :account_reset_request, dependent: :destroy
   has_many :phone_configurations, dependent: :destroy, inverse_of: :user
   has_one :email_address, dependent: :destroy, inverse_of: :user
-  has_many :webauthn_configurations, dependent: :destroy
+  has_many :webauthn_configurations, dependent: :destroy do
+    def selection_presenters
+      if any?(&:mfa_enabled?)
+        [TwoFactorAuthentication::WebauthnSelectionPresenter.new(first)]
+      else
+        []
+      end
+    end
+  end
```

Comment:
What do you think of moving this to its own class? We try to keep the User model lean.

Reply:
It wouldn't work. It needs to be on the webauthn_configurations object. If we decide to have a separate choice for each hardware token, then we can drop this and move it into the configuration class.